src/middlewares/userValidations.js

```diff
@@ -45,6 +45,34 @@ const emailExists = (req, res, next) => {
   });
 };
 
+/**
+ * @function
+ * @description Check if user email exist, password correct and verified
+ * @param {object} req - Resquest object
+ * @param {object} res - Response object
+ * @param {object} next
+ * @returns {object} JSON response
+ */
+const loginData = (req, res, next) => {
```

Comment:
You can rename this to be more descriptive about what the function does
```suggestion
const validateLogin = (req, res, next) => {
```

Reply:
Thanks. Changes effected